qtpydocking/dock_container_widget.py

```diff
@@ -227,8 +228,14 @@ def drop_into_container(self,
         insert_param = dock_area_insert_parameters(area)
         floating_dock_container = floating_widget.dock_container()
 
-        new_dock_areas = floating_dock_container.findChildren(
-            DockAreaWidget, '', Qt.FindChildrenRecursively)
+        api = os.environ['QT_API']
+        if api == 'pyside2':
+            new_dock_areas = floating_dock_container.findChildren(
+                DockAreaWidget, '')
+        else:
+            new_dock_areas = floating_dock_container.findChildren(
```

Comment:
Since we need to use the API check to get `findChildren` to work multiple times, would you mind making a new method in `qtpydocking.util` that does this check only once? e.g., `util.find_children(parent, cls, name='', options=Qt.FindChildrenRecursively)`

Reply:
Added. Much more cohesive now. :)